epic_pfrich_sl_only.xml

```diff
@@ -0,0 +1,121 @@
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
```

Comment:
Remove this file and use configuration/.

Reply:
removed